graphql-dgs-codegen-gradle/src/test/java/com/netflix/graphql/dgs/CodegenGradlePluginTest.java

```diff
@@ -84,7 +84,7 @@ public void sourcesGenerated() {
         // Verify the result
```

Comment:
This test suite is more for testing gradle plugin related functionality rather than features. You can add tests for this in the core module (we already have an extensive test suite there that you can add to) and verify using the command line with this option passed in.

Reply:
Hi again @srinivasankavitha!

Thanks for pointing this out. For some reason, I completely missed those tests. And thankfully, they proved that the task wasn't as trivial.

I ended up changing the `toString` implementations drastically to allow filtering _null_ values. They use a linked map to add the keys and values, and based on that `omitNullInputFields` a filter is added or not to remove the _null_s.

I also changed the Java tests quite a bit. Where they were checking for the generated `toString` code, I now instantiate that class, call `toString`, and assert on the produced output instead. I'm still working on the code that creates a class loader for that (`fun compileAndGetConstructor`) but wanted to validate if this makes sense. I couldn't do the same to Kotlin, unfortunately, as the generated code is not being compiled in the tests.

Let me know what you think, and thanks once again!

Reply:
I pushed one last change to use an in-memory filesystem in `compileAndGetConstructor`.